src/Controller/DownloadController.php

```diff
@@ -19,8 +20,15 @@ public function learningMaterialAction(
         $token,
         LearningMaterialManager $learningMaterialManager,
         IliosFileSystem $iliosFileSystem,
-        Request $request
+        Request $request,
+        Config $config
     ) {
+        if ($config->get('learningMaterialsDisabled') === true) {
+            return new Response(
```

Comment:
a 204 response code may be more appropriate here. let's talk about it tomorrow.

https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/204

Reply:
204 is usually sent with no content, I added some text here to ensure it was obvious what was happening.